src/main/java/seedu/address/model/student/Note.java

```diff
@@ -9,8 +9,8 @@
 public class Note {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Note should be 200 characters or less";
-    public static final String VALIDATION_REGEX = ".{0,200}";
+            "Note should be 500 characters or less";
+    public static final String VALIDATION_REGEX = ".{0,500}";
```

Comment:
Good change to accommodate new Notes column, as notes will probably be longer now.